gnovm/pkg/gnolang/nodes.go

```diff
@@ -1671,6 +1779,17 @@ func (sb *StaticBlock) GetIsConst(store Store, n Name) bool {
 	}
 }
 
+func (sb *StaticBlock) GetIsConstAt(store Store, path ValuePath) bool {
+	for {
+		if path.Depth == 1 {
+			return sb.getLocalIsConst(path.Name)
+		} else {
+			sb = sb.GetParentNode(store).GetStaticBlock()
```

Comment:
Please merge this up with other instances of this "traversing" algorithm :)

See-also https://github.com/gnolang/gno/pull/2429#discussion_r1735249298

Reply:
I've consolidated the normal traversal case here https://github.com/gnolang/gno/pull/2731/commits/96984614cb6306cea058a13e507778996f772ea3